packages/popover/test/Overlay.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { render, fireEvent, cleanup } from 'react-testing-library'
-import Overlay from '../src/overlay'
+import Overlay from '../src/Overlay.js'
```

Comment:
@dgiraldo313 & @sdalonzo, I've used the suggested change feature to omit `.js`. Once tested locally, I will apply the change to this thread and move forward with the merge.

```suggestion
import Overlay from '../src/Overlay'
```